bootstrap/pkg/utils/gcputils.go

```diff
@@ -71,64 +59,3 @@ func BuildConfigFromClusterInfo(ctx context.Context, cluster *containerpb.Cluste
 	}
 	return config, nil
 }
-
-// Helper function for the rest.Config retrieval.
-func BuildConfigForGcp(ctx context.Context, project string, loc string, cluster string) (*rest.Config, error) {
-	log.Infof("Getting rest.Config with GCP for %v/%v/%v", project, loc, cluster)
-	clusterpb, err := GetClusterInfo(ctx, project, loc, cluster)
-	if err != nil {
-		return nil, err
-	}
-	return BuildConfigFromClusterInfo(ctx, clusterpb)
-}
-
-// Create a config that serves as kubeconfig.
-func CreateKubeconfig(ctx context.Context, project string, loc string, cluster string,
```

Comment:
please don't delete this.  I need it to fix #2586

Reply:
Done